spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SpannerStatementQueryExecutor.java

```diff
@@ -167,19 +168,30 @@ private static void buildWhere(PartTree tree, SpannerPersistentEntity<?> persist
 				StringJoiner andStrings = new StringJoiner(" AND ");
 
 				orPart.forEach(part -> {
-
 					String segment = part.getProperty().getSegment();
 					String tag = "tag" + tags.size();
 					tags.add(tag);
 					String andString = persistentEntity.getPersistentProperty(segment)
 							.getColumnName();
+					String sortedTag = "@" + tag;
```

Comment:
Why are you calling this `sortedTag`? How is this related to sorting?

Reply:
I will rename